components/application-connector/README.md

```diff
@@ -105,11 +103,11 @@ Although you are on your local machine, you can make calls such as `curl http://
 ```bash
 telepresence --namespace kyma-system --swap-deployment metadata:metadata --run-shell
 ```
-4. Every Kubernetes pod has `/var/run/secrets` mounted. The Kubernetes client uses it in the Application Connector services. It is hardcoded. By default, telepresence copies this directory. It stores the directory path in `$TELEPRESENCE_ROOT`, under telepresence shell. It unwinds to `/tmp/tmp...`. You need to move it to `/var/run/secrets`, where the service expects it. Create a symlink:
+4. Every Kubernetes Pod has `/var/run/secrets` mounted. The Kubernetes client uses it in the Application Connector services. It is hardcoded. By default, telepresence copies this directory. It stores the directory path in `$TELEPRESENCE_ROOT`, under telepresence shell. It unwinds to `/tmp/tmp...`. You need to move it to `/var/run/secrets`, where the service expects it. Create a symlink:
  ```bash
 sudo ln -s $TELEPRESENCE_ROOT/var/run/secrets /var/run/secrets
 ```
-5. Use the `make build` and then the `./metadata` commands, and now all the Kubernetes services that call metadata access this process. The process runs locally on your machine. Use the same command to run different Application Connector services like Proxy or Events.
+5. Run `make build`, then `./metadata` to give all  Kubernetes services that call the Metadata Service to access this process. The process runs locally on your machine. Use the same command to run different Application Connector services like Proxy or Events.
```

Comment:
This instruction is no longer valid. During consolidation phase we removed makefiles. Command are executed directly in Jenkins job.